samcli/commands/_utils/custom_options/option_help.py

```diff
@@ -0,0 +1,28 @@
+"""
+Custom click option for customizing help text.
+"""
+
+import click
+
+from samcli.lib.utils.colors import Colored
+
+
+class RequiredOptionColorTextOption(click.Option):
+    def __init__(self, *args, **kwargs):
+        super(RequiredOptionColorTextOption, self).__init__(*args, **kwargs)
+        self.colored = Colored()
+
+    def get_help_record(self, ctx):
+        option, help_text = super(RequiredOptionColorTextOption, self).get_help_record(ctx=ctx)
+        return (self.colored.yellow(option), self.colored.yellow(help_text)) if self.required else (option, help_text)
```

Comment:
I think the idea is great (especially consider the number of parameters), although a little on the fence whether yellow is ideal. It's typically used to convey warnings and mutations, but at the same time it's a common accent color in SAM CLI. 🤔

Reply:
my other thinking is we could make it bold or italic?

Reply:
Italic support is sketchy in some terminals/fonts. Making only the option bold but regular text color might be nice as well. To be clear I like colors especially if we can define color guidelines to avoid inconsistent "over-saturation".